pkg/tests/apis/provisioning/testdata/text-options.json

```diff
@@ -0,0 +1 @@
+../../../../../devenv/dev-dashboards/panel-text/text-options.json
```

Comment:
Yes: this is a symlink. That means that this is effectively just the file it points to. This is the same thing we do with the all-panels file.